tests/contrib/sqlite3/test_sqlite3.py

```diff
@@ -31,6 +32,22 @@ def setUp(self):
     def tearDown(self):
         unpatch()
 
+    def test_service_info(self):
+        tracer = get_dummy_tracer()
+        backup_tracer = ddtrace.tracer
```

Comment:
I've created a decorator to do that: https://github.com/DataDog/dd-trace-py/blob/fe1b4ece7513fd305ab0b4713bf8508a0c4af3af/tests/contrib/httplib/utils.py#L6-L16

You can use it that way: https://github.com/DataDog/dd-trace-py/blob/fe1b4ece7513fd305ab0b4713bf8508a0c4af3af/tests/contrib/httplib/test_httplib.py#L323-L324

Now that you're here, you can:
* move the `override_global_tracer` from `dd-trace-py/tests/contrib/httplib/utils.py` to `dd-trace-py/tests/util.py`
* update `httplib` import
* use it in this test

In this way, we'll make it test-wide available. Thank you!